core/src/main/java/io/seata/core/exception/AbstractExceptionHandler.java

```diff
@@ -124,7 +130,12 @@ public void onException(T request, S response, Exception rex) {
             callback.execute(request, response);
             callback.onSuccess(request, response);
         } catch (TransactionException tex) {
-            LOGGER.error("Catch TransactionException while do RPC, request: {}", request, tex);
+            if (Objects.equals(LockKeyConflict, tex.getCode()) || Objects.equals(LockKeyConflictFailFast, tex.getCode())) {
```

Comment:
LockKeyConflictFailFast的场景，是不是“尝试获得全局锁的时候，发现全局事务的决议是回滚，放弃加锁，直接快速失败”？我改成了这样。
```java
    public <T extends AbstractTransactionRequest, S extends AbstractTransactionResponse> void exceptionHandleTemplate(Callback<T, S> callback, T request, S response) {
        try {
            callback.execute(request, response);
            callback.onSuccess(request, response);
        } catch (TransactionException tex) {
            if (Objects.equals(LockKeyConflict, tex.getCode())) {
                LOGGER.info("this request cannot acquire global lock, you can let Seata retry by setting config [{}] = false or manually retry by yourself. request: {}",
                        ConfigurationKeys.CLIENT_LOCK_RETRY_POLICY_BRANCH_ROLLBACK_ON_CONFLICT, request);
            } else if (Objects.equals(LockKeyConflictFailFast, tex.getCode())) {
                LOGGER.info("this request cannot acquire global lock, decide fail-fast because LockStatus is {}. request: {}",
                        LockStatus.Rollbacking, request);
            } else {
                LOGGER.error("Catch TransactionException while do RPC, request: {}", request, tex);
            }
            callback.onTransactionException(request, response, tex);
        } catch (RuntimeException rex) {
            LOGGER.error("Catch RuntimeException while do RPC, request: {}", request, rex);
            callback.onException(request, response, rex);
        }
    }

```